edit-post/store/effects.js

```diff
@@ -90,6 +89,28 @@ const effects = {
 		const message = action.mode === 'visual' ? __( 'Visual editor selected' ) : __( 'Code editor selected' );
 		speak( message, 'assertive' );
 	},
+	INIT( _, store ) {
+		// Select the block settings tab when the selected block changes
+		subscribe( onChangeListener(
+			() => get( select( 'core/editor' ).getSelectedBlock(), 'uid' ),
```

Comment:
I like this pattern, to execute something after a change in another store. But our effects code is already huge and not easy to organize, this pattern is similar to effects but for other stores so I think it may also become huge when we increase its usage.
Could we have another file with this operations and here we just call a function from that file passing the store to it?

Reply:
I was hesitant here, using another file would mean that these things are not side effects, but they are, just expressed slightly differently, I was hesitant to introduce another "concept".

Reply:
It's still better to find it here than in the store file :)